limiter.go

```diff
@@ -0,0 +1,62 @@
+package go_redis_ratelimit
+
+import (
+	"errors"
+
+	"github.com/go-redis/redis/v8"
+
+	"github.com/bringg/go_redis_ratelimit/algorithm"
+
+	_ "github.com/bringg/go_redis_ratelimit/algorithm/all"
+)
+
+const (
+	DefaultPrefix = "limiter"
+)
```

Comment:
just a nit, but having this one line const, makes code look cleaner, unless you have multiple const definitions
same for other things, like `var` and `type`